app/code/Magento/MediaGallery/Model/ResourceModel/Keyword/SaveAssetLinks.php

```diff
@@ -28,25 +30,33 @@ class SaveAssetLinks
      */
     private $resourceConnection;
 
+    /**
+     * @var GetAssetsKeywordsInterface
+     */
+    private $getAssetsKeywordsInterface;
```

Comment:
I propose to use a shorter and more precise property naming
```suggestion
    private $getAssetsKeywords;
```